render/renderer.go

```diff
@@ -196,30 +201,48 @@ func (r *Renderer) lookup(name string) (string, error) {
 	}
 
 	if _, isThunk := meta.Value().(*PrepareThunk); isThunk {
-		log.WithField("proxy-reference", vertexName).Warn("node is unresolvable")
+		log.WithField("proxy-reference",
```

Comment:
this and the Sprintf call below are nested a little weird. Could you change it to the…

```go
x(
    a,
    b,
).c(
    d(
        e,
        f,
    )
)
```

… style?